packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -79,6 +79,7 @@ const Dropdown: React.FunctionComponent<IDropdownProps & ThemeProps<DefaultTheme
   const nextItemsHashRef = useRef<Record<string, unknown>>({});
   const containsMultiselectRef = useRef(false);
   const itemSearchRegistry = useRef([]);
+  const [dropdownType, setDropdownType] = useState<DROPDOWN_TYPE>('');
```

Comment:
wasn't able to come up with a more elegant means of knocking-out base dropdown functionality. Ideally, this would build from `Combobox` -> `Autocomplete` -> `Multiselect` and be cleaner. But ... history.